taxonomies/ingredients.txt

```diff
@@ -76663,6 +76663,7 @@ bg:краставица, краставици
 ca:cogombre
 cs:Okurek
 da:agurk, agurker
+de:Gurke, Gurken
```

Comment:
```suggestion
de:Gurken, Gurke
```
Categories are usually plural first, with the singular alias following next

Reply:
This is not a category, this is an ingredient :)

Reply:
This is not a category, this is an ingredient :)